### DISCUSSION TITLE
More information feedback

### DISCUSSION THREAD

**Discussion by theDanielJLewis** [theDanielJLewis/podcasting2.org#1] [General]
I received this feedback directly from someone:

> I admit that I have not been following Podcasting 2.0 very much and don't know its intricacies. The way I see it is what I've been doing is working for me, so why change it.
>
> After today's PodNews, I decided to check out your [podcasting2.org](http://podcasting2.org/) website (clever domain, BTW.) Maybe you and James are afflicted with the "curse of knowledge," but after reading through the site, I'm no closer to understanding what Podcasting 2.0 is or why I should care.
> 
> You list podcast namespace benefits but don't explain any of them. For example, "Expose more information about your podcast, like people, location, and more." What does that mean? Or "Deliver new episodes more quickly to your podcast followers." How? And why is this a benefit?
>
> I consider myself a somewhat techie person, but I still don't get the hype. The website needs much more information, written in a less jargony way, if you expect it to bring people into the fold.

**Comment by jamescridland** [comment]
I think it’s fair enough, and that we dive into the detail before producing an overview. Worth thinking on that and how we’d do it.

**Comment by theDanielJLewis** [comment]
I'm thinking a sort of "Who? What? When? Where? How?" approach could be a good start. I'm working on some of that right now for the front page.

**Reply by theDanielJLewis** [reply]
Here's a draft in the `dev` branch: https://github.com/theDanielJLewis/podcasting2.org/blob/dev/src/app/page.tsx

**Comment by Dwev** [comment]
Hi guys, I hope I'm welcome to join the conversation here. 😁 

I was actually coming to say something similar, that for a listener, they are non wiser about *why* they should care about PC2.0 (is that the right abbreviation?)

They are shown a list of apps, but they will have a harder time changing from what they are comfortable with, without good reason.  

Maybe before listing all the apps, it would be good to describe the benefit to the listener.  

Instead of the `<podcast:podroll>` format for them, have the benefit of "recommendations from the podcaster's you already love".

Transcripts not only benefit hearing-impaired people, but also allow anyone to consume a podcast, even if they can't hear it at the time. You can also find something specific in a transcript by using a page search.

On a more meta level, it supports the idea of there being a level playing field. That could be a good enough reason for people to change from the monolithic Apple and Spotify, to more grass-roots, OSS, future-thinking apps.

Personally, I'm looking at the potential for audiobooks, as the distribution power for these is very centralized. How many people can name more than Audible for audiobooks?  How many independent authors would like to avoid the big A for publishing not only their print / e-books, but audiobooks as well.  There is a massive potential to quickly have feature parity with Audible, in supporting PC2.0 apps.

**Reply by theDanielJLewis** [reply]
> Hi guys, I hope I'm welcome to join the conversation here. 😁

Absolutely! That's why this is open source. 😃

This is great feedback! Would you be interested in writing up some of these audience-focused descriptions for each feature?

**Reply by Dwev** [reply]
I'll give it a go!

Not sure I can put it into a PR just yet, but maybe a simple table will be enough?

**Reply by theDanielJLewis** [reply]
That's okay. You could simply start a new discussion or issue with all of your parts.

**Reply by Dwev** [reply]
Cool, thanks Daniel.  I liked your recent episode where you talked about cross-platform comments.  I'm guessing this is based on the `socialInteract` tag, specifically around ActivityPub?  The way you explained it was great, and I noted you were cagey about saying "ActivityPub" since that already brings expectations.

The listener doesn't care about the mechanism, they only care that they don't have to leave their app to comment, and they still benefit from others comments, regardless of which apps those listeners used.  

It's like you can use different browsers to display a forum, and the content is (mostly) the same regardless of the browser. In this metaphor, the browser is the podcast app. The app should be agnostic to the content, including comments etc.

**Reply by theDanielJLewis** [reply]
Yes, ActivityPub frustrates me, especially since it leaves the podcaster with little to no control, ownership, or portability. But I started some discussion threads on the Podcast Namespace repo to suggest some solutions and get the conversations rolling to solve those problems.

- https://github.com/Podcastindex-org/podcast-namespace/discussions/612
- https://github.com/Podcastindex-org/podcast-namespace/discussions/611
- https://github.com/Podcastindex-org/podcast-namespace/discussions/610

**Reply by Dwev** [reply]
Sorry it's an image, was doing this from my iPad.  I can convert it MD when it's more complete, but I wanted to share an initial draft.

What do you think of this concept?

![image](https://github.com/theDanielJLewis/podcasting2.org/assets/46626050/6453d99f-f467-4d43-a6b2-757b0a59fd54)

**Reply by Dwev** [reply]
@theDanielJLewis @jamescridland Any thoughts on the concept above?

**Reply by Dwev** [reply]
So, I forked the repo, and had a go and making some changes, but before I do too much, I want to make sure I'm working in the right direction. I added the PC 20 logo to the header, and added an accordion type panel (`<details><summary>`) and more of those nested for each tag. I am not in love with the design, so this is just an example of functionality.

The idea is to give listeners ("fans") contextual information that describes the benefits of the tags on the same page they will see the app list. So when they click on an app and see "podroll" they know what that means, or they can filter apps by the tag/benefit (this is a feature of the Podcastindex.org site that's I miss here).

@theDanielJLewis you mentioned that you were planning to move the working db from podcast index, so you can use it for your other site.  Maybe we can add a column in the tag table that describes the benefits (and other columns for how hosts and app devs can support like in the table above), so it can be maintained a bit easier.

Let me know what you think.

![image](https://github.com/theDanielJLewis/podcasting2.org/assets/46626050/7257c798-7747-4a37-8cd9-86d8d0720ded)

**Reply by theDanielJLewis** [reply]
I like the idea, but I don't think that's the right approach. Visually or conceptually. I think we should be benefits-first.

For example, most podcasters have no idea what "podroll" is, so they won't know why they should even read about it.

**Reply by jamescridland** [reply]
Agree. Podroll is a stupid name. It’s fine for a tag, but it shouldn’t be visible to any human being. The same goes with much of the tags we have. I hope they’re not brands that we use. 

podcastnamespace.org was my attempt to do similar,  @Dwev - trying to convey the functionality in one sentence.

What frustrates me a bit is not understanding enough about how to get a local instance running - I can edit existing pages but can’t really write new sections, without screwing something up and taking the entire website down. I wish I could do more to the p2.0 website whenever I have the urge.

**Reply by theDanielJLewis** [reply]
@jamescridland (and others), I'll add to this repo's readme with better instructions on starting a local dev.

I also discovered something that will make it easier to write mostly normal HTML content without having to worry about all the `className` attributes.

**Comment by Dwev** [comment]
One thing I'd recommend changing is the collective noun of  "Audience" to the singular noun of "Listener". 

When someone comes to the site, they automatically try to categorize themselves based on the options given.  Devs are no problems, and neither are podcasters, but an individual who consumes podcasts will think "am I an audience?" which adds a little friction as soon as they get here. "Audience" suggests that every listener is just part of one big group and doesn't have individual preferences or tastes. "Audience" is framed from the point of view of the performer.

If you say "for Listeners" they will immediately identify "hey, I listen to podcasts" and it's easy to categorize themselves as it's from their own point of view. Then they will click to find out "how will this benefit me?"

**Reply by theDanielJLewis** [reply]
Yeah, but podcasts have always been video, too. That's why I went for the cross-medium term "audience."

**Reply by Dwev** [reply]
Ah, good point. Listener is not universal enough for the different mediums. 🤔 

The best I can come up right now is "audience member" but that's a bit clunky.

Some alternatives for consideration:
- Subscriber
- Follower
- Consumer (yuk)
- Individual (may not be specific enough as podcasters and devs are also individuals)
- Fan (a nod to Sam 😉)
- Listener / Viewer (could exclude non AV mediums)

**Reply by theDanielJLewis** [reply]
"Fan" could work. "Listener/viewer" can work fine within body text.
